slt/standard/functions/scalars/string/left.slt

```diff
@@ -0,0 +1,48 @@
+# LEFT
+
+query T
+SELECT left('hello', 3);
+----
+hel
+
+query T
+SELECT left('hello', 5);
+----
+hello
+
+query T
+SELECT left('hello', 10);
+----
+hello
+
+query T
+SELECT left('hello', 0);
+----
+(empty)
+
+query T
+SELECT left('hello', -5);
+----
+(empty)
+
+query T
+SELECT left('hello', -3);
+----
+he
```

Comment:
Add a test case `SELECT left('hello', -8);` to ensure still get an empty string if the abs value is greater than string length.